VaticanBAV/et/BAVet132.xml

```diff
@@ -0,0 +1,412 @@
+                <locus from="1" to="2"/>
+              </item>
+                                <item xml:id="q2" n="1">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="3" to="12"/>
+                                </item>
+                                <item xml:id="q3" n="2">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="13" to="22"/>
+                                </item>
+                                <item xml:id="q4" n="3">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="23" to="30"/>
+                                </item>
+                                <item xml:id="q5" n="4">
+                                  <dim unit="leaf">9</dim>
+                                  <locus from="31" to="39"/>
+                                </item>
+                                <item xml:id="q6" n="5">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="40" to="49"/>
+                                </item>
+                                <item xml:id="q7" n="6">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="50" to="59"/>
+                                </item>
+                                <item xml:id="q8" n="7">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="60" to="69"/>
+                                </item>
+                                <item xml:id="q9" n="8">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="70" to="79"/>
+                                </item>
+                                <item xml:id="q10" n="9">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="80" to="89"/>
+                                </item>
+                                <item xml:id="q11" n="10">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="90" to="99"/>
+                                </item>
+                                <item xml:id="q12" n="11">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="100" to="109"/>
+                                </item>
+                                <item xml:id="q13" n="12">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="110" to="117"/>
+                                </item>
+                                <item xml:id="q14" n="13">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="118" to="127"/>
+                                </item>
```

Comment:
q14 needs to be deleted to achieve the total number of 14 quires and 128(+2) folios.